### DISCUSSION TITLE
Need help to add icon and text inside node

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#399] [Q&A] [answered]
I am using GLSP's vscode client(https://github.com/eclipse-glsp/glsp-vscode-integration),

I want add icon + text inside node

If we add parameters(additional info related to node or type of icon or hyperlinks ) to node object at server side, can we access them at client side while generating SVG?

**Screenshot:**
![image](https://user-images.githubusercontent.com/59145404/134341487-8c4f747b-728f-47b4-b2dd-b8e8934f04a7.png)

**Comment by planger** [comment, marked as answer]
You can put arbitrary `args` into the `GModelElements` when constructing the `GGraph`. On the client you can then react to them in the views when generating the SVG and e.g. set an icon or a link.
